test/test_smoke.py

```diff
@@ -39,6 +41,111 @@ def test_smoke_has_expected_images_centos(build_container):
                 assert f"{distro} type:{type_} arch:{arch}" in output
 
 
+def test_smoke_has_expected_images_almalinux_8(build_container):
```

Comment:
Do we have a similarly comprehensive test for rhel? And if not, should we create one for rhel first maybe?